centreon/www/class/centreonGraph.class.php

```diff
@@ -399,7 +408,7 @@ private static function quote($elem)
      */
     private static function vquote($elem)
     {
-        return "'" . substr($elem, 1, strlen($elem) - 1) . "'";
+        return  substr($elem, 1, strlen($elem) - 1) ;
```

Comment:
Same as previous
```suggestion
        return "'" . substr($elem, 1, strlen($elem) - 1) . "'";
```

Reply:
i test it no brecking change
i did that to update a SQL query because it Doesn't work because the 'bindValue' function will enclose the value in single quotes.